pkg/config/config.go

```diff
@@ -280,7 +280,9 @@ func InitConfig(config Config) {
 	config.BindEnvAndSetDefault("remote_configuration.enabled", false)
 	config.BindEnvAndSetDefault("remote_configuration.key", "")
 	config.BindEnv("remote_configuration.api_key")
-	config.BindEnv("remote_configuration.rc_dd_url", "")
+	config.BindEnv("remote_configuration.rc_dd_url")
+	config.BindEnvAndSetDefault("remote_configuration.rc_no_tls", false)
```

Comment:
Just a simple question: what does `rc` means here ? 

If it is for `remote_configuration` then is it expected to have `remote_configuration.rc_no_tls` instead of just `remote_configuration.no_tls` for example ? I think it's the convention for `dd_url` but I don't know for the others options tbh so I'm just asking.

Reply:
tbh I've copied the pattern for `dd_url` as other products were doing something similar and ported that to other similar options for consistency. I can remove the prefix if there's no reason for it.

Reply:
AFAIK this pattern is only present for `dd_url` so I don't think there is any reason to keep `rc` as it is under the `remote_configuration` config. I don't have a strong opinion on it tbh